dnn/data/converters/get_purdue.py

```diff
@@ -0,0 +1,97 @@
+import json
+import xml.dom.minidom as minidom
+
+import requests
+from absl import app, flags
+from absl.flags import FLAGS
+from tqdm import tqdm
+import os
+import time
+
+flags.DEFINE_string('input', '../Purdue Dataset/label.json',
+                    'the path for input ROCO dataset (please unzip by yourself)')
+flags.DEFINE_string('output', '../Purdue Dataset',
+                    'the path for input ROCO dataset (please unzip by yourself)')
+
+
+def main(_argv):
```

Comment:
If `_argv` is unused, this should be `_` instead. Also, The reason I prefer `argparse` is because I don't like global variables like `flags` here. But this is more of a design choice, and the arguments seem reasonable, so I will leave it up to you whether to use `absl` or `argparse`.

Reply:
I prefer absl. :))))))))))))))))))))))))))))))))))))))))))))))))))))